### ISSUE TITLE
cannot install on laravel 9

### ISSUE THREAD

**Issue by robertnicjoo** [PHPOffice/PhpSpreadsheet#3057]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?

Install on laravel 9

### What is the current behavior?

installation error

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
composer require phpoffice/phpspreadsheet
```

```php
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires phpoffice/phpspreadsheet ^1.24 -> satisfiable by phpoffice/phpspreadsheet[1.24.0, 1.24.1].
    - phpoffice/phpspreadsheet[1.24.0, ..., 1.24.1] require psr/simple-cache ^1.0 || ^2.0 -> found psr/simple-cache[1.0.0, 1.0.1, 2.0.0, 2.x-dev] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require phpoffice/phpspreadsheet:*" to figure out if any version is installable, or "composer require phpoffice/phpspreadsheet:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```
If this is an issue with reading a specific spreadsheet file, then it may be appropriate to provide a sample file that demonstrates the problem; but please keep it as small as possible, and sanitize any confidential information before uploading.

### What features do you think are causing the issue

- [ ] Reader
- [ ] Writer
- [ ] Styles
- [ ] Data Validations
- [ ] Formula Calculations
- [ ] Charts
- [ ] AutoFilter
- [ ] Form Elements

### Which versions of PhpSpreadsheet and PHP are affected?

```php
php: 8.1.x
phpspreadsheet: latest version
```

**Comment by MarkBaker** [comment]
Either install the latest master of PhpSpreadsheet, which will work with psr/simple-cache 3; or downgrade your psr/simple-cache to version 2

**Comment by robertnicjoo** [comment]
> Either install the latest master of PhpSpreadsheet, which will work with psr/simple-cache 3; or downgrade your psr/simple-cache to version 2

I suppose `composer require phpoffice/phpspreadsheet` should installk latest master version as there is no version defined in command. isn't that right?

**Comment by MarkBaker** [comment]
No: If you do a composer require without specifying a version, then composer will pick the latest tagged release; it will only install the latest commit against master if there is no tagged release.

You'll need to specify `dev-master` as the version; and may need to specify `--dev` for to override the minimum stability

**Comment by skeemer** [comment]
I hope we can use an official release, but for now this is the command.

```
composer require phpoffice/phpspreadsheet:dev-master
```

**Comment by MarkBaker** [comment]
Resolved in release `1.25.0`
